### PR TITLE
FIx support for quiet option in BuildImageCmd

### LINKED ISSUE #1246: BuildImageCmd fails when withQuiet option is used
When building docker image with `quiet` option deamon doesn't return anything except image id in the following format:
````console
sha256:691f739c182efcf4bb7b1aca822db6aae93f642f7961a5dd2c82f5588655df97
````

As for now, `com.github.dockerjava.api.model.BuildResponseItem` doesn't support such scenario and exception is thrown even image was successfully built.

This issue was reported in [gradle-docker-plugin](https://github.com/bmuschko/gradle-docker-plugin/issues/859) repo.

### PR DESCRIPTION
**Pull request by orzeh**
Fix #1246

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/1247)

### CODE REVIEW

**Comment by KostyaSha** [comment]
could you rebase?

**Comment by orzeh** [comment]
Rebased.

File: docker-java/src/test/java/com/github/dockerjava/cmd/BuildImageCmdIT.java [hunk 1]
```diff
@@ -313,6 +313,21 @@ public void cacheFrom() throws Exception {
 
     }
 
+    @Test
+    public void quiet() {
```
Comment (bsideup) [inline docker-java/src/test/java/com/github/dockerjava/cmd/BuildImageCmdIT.java]: this test does not seem to test much, could you please verify `getImageId` and `isBuildSuccessIndicated()`?

Comment (orzeh) [reply docker-java/src/test/java/com/github/dockerjava/cmd/BuildImageCmdIT.java]: Sure, will do.

Comment (orzeh) [reply docker-java/src/test/java/com/github/dockerjava/cmd/BuildImageCmdIT.java]: @bsideup I've opened #1318 wherein I've rebased on master and added additional checks. Please review if possible. Please note that `isBuildSuccessIndicated()` is called internally when getting image id.

**Pull request closed without merging**